data/pigui/modules/map-sector-view.lua

```diff
@@ -18,87 +19,226 @@ local colors = ui.theme.colors
 local icons = ui.theme.icons
 local hideSectorViewWindows = false
 
-local mainButtonSize = Vector2(32,32) * (ui.screenHeight / 1200)
+local mainButtonSize = ui.rescaleUI(Vector2(32,32), Vector2(1600, 900))
 local mainButtonFramePadding = 3
-local function mainMenuButton(icon, selected, tooltip, color)
-	color = color or colors.white
-	return ui.coloredSelectedIconButton(icon, mainButtonSize, selected, mainButtonFramePadding, colors.buttonBlue, color, tooltip)
+
+local font = ui.fonts.pionillium.medlarge
+local MAX_SEARCH_STRINGS_VISIBLE = 15
+local textIconSize = nil
+local edgePadding = nil
+
+local function setAlpha(c, a)
+	return Color(c.r, c.g, c.b, a)
+end
+-- all colors, used in this module
+local svColor = {
```

Comment:
I was surprised that there is no mechanism in the С++ side to read data from the theme file (or is there?)

Reply:
@Gliese852 I'd be surprised if there was, since the theme file was implemented for the pi-gui switch (I think), and we decided to kill off C++ UI, which was started in 2014. Thus: no work on C++ UI side, all UI work to kill off C++

Reply:
@impaktor  But it turned out that the theme settings are needed not only for UI, for example, the colors of the grid in maps, the colors of the orbits should be combined with the colors of the icons, e.t.c.